nuxeo-runtime/nuxeo-launcher-commons/src/main/java/org/nuxeo/launcher/config/ConfigurationGenerator.java

```diff
@@ -375,6 +385,24 @@ protected Properties getStoredConfig() {
         }
     };
 
+    public static File[] getJarFilesFromPattern(File dir, String pattern) {
+        final Pattern jarPattern = Pattern.compile(pattern);
+
+        File[] foundJarFiles = dir.listFiles(new FileFilter() {
+            @Override
+            public boolean accept(File f) {
+                return jarPattern.matcher(f.getName()).matches();
+            }
+        });
```

Comment:
This can use a lambda:

        File[] foundJarFiles = dir.listFiles(f -> jarPattern.matcher(f.getName()).matches());